pandas/core/dtypes/common.py

```diff
@@ -220,6 +220,10 @@ def is_categorical(arr):
     return isinstance(arr, ABCCategorical) or is_categorical_dtype(arr)
 
 
+def is_range(arr):
```

Comment:
This is only called in ``get_dtype_kinds``, see below, so I could have just used ``isinstance(arr, ABCRangeIndex)``. But I tried to be coherent with ``is_categorical`` and ``is_sparse``... what exactly is a "type"?!